### ISSUE TITLE
Add custom class weights to CE and Dice Loss functions

### ISSUE THREAD

**Issue by MattAWard** [MIC-DKFZ/nnUNet#2849]
When trying to train an anatomical brain model the smaller structures (like the cochlea) are being ignored as they carry little weight in the overall loss function. As such, I would like to impose a custom weight vector which I have generated by taking the log of 1/(relative volume) of each structure over my set of training data:

[0.71, 0.71, 3.94, 9.92, 9.91, 5.24, 5.22, 8.82, 8.82, 7.72, 7.58, 7.57]

 However, I do not know where to specify this. Can I just make a custom trainer in a new .py file in the variants section of the nnUNetTrainer folder which calls this vector in some way, or do I need to somehow edit the CE and Dice loss functions more directly?

**Comment by constantinulrich** [comment]
Hi, exactly: make a new trainer and overwrite the built loss function would be the way to go. 
Best
Constantin
